### PR TITLE
Find and replace malfunction

### PR DESCRIPTION
**Pull request by Ashersam**
fix for #158  Added windows reload and loader with disabled button `save changes` on zero find.

### CODE REVIEW

**Comment by joelthe1** [comment]
How did we test this so far?

**Comment by Ashersam** [comment]
> How did we test this so far?

I tested manually quite many times. Didn't throw any error. I've asked Vipin also to do the same

**Comment by Ashersam** [comment]
> > How did we test this so far?
> 
> I tested manually quite many times. Didn't throw any error. I've asked Vipin also to do the same

issue still exists its not with db manupulation or loader. I think the issue with the code some. Vipin found out one pattern that throws error in first iteration itself

**Comment by Ashersam** [comment]
> > > How did we test this so far?
> > 
> > 
> > I tested manually quite many times. Didn't throw any error. I've asked Vipin also to do the same
> 
> issue still exists its not with db manupulation or loader. I think the issue with the code some. Vipin found out one pattern that throws error in first iteration itself

@joelthe1 Added code fix for manipulation it was a small logical change in code and added loader test by both of us.

File: src/components/Search.js [hunk 1]
```diff
@@ -132,7 +136,7 @@ class SearchModal extends React.Component {
     db.get(AutographaStore.bookId.toString()).then((doc) => {
         if (AutographaStore.replaceOption === "chapter") {
             for (var c in replacedChapter) {
-                var verses = doc.chapters[AutographaStore.chapterId-1].verses
+                var verses = (doc.chapters[AutographaStore.chapterId-1]).verses
```
Comment (joelthe1) [inline src/components/Search.js]: Was this the logical error you were talking about?

Comment (Ashersam) [reply src/components/Search.js]: nope this is just auto-formatting done by prettier

Comment (Ashersam) [reply src/components/Search.js]: @joelthe1 I have changed loader text to gif

**Comment by joelthe1** [comment]
This fixes #158

**Pull request merged**